DependencyInjection/Configuration.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+declare(strict_types=1);
+
+namespace PetitPress\GpsMessengerBundle\DependencyInjection;
+
+use Symfony\Component\Config\Definition\Builder\TreeBuilder;
+use Symfony\Component\Config\Definition\ConfigurationInterface;
+
+final class Configuration implements ConfigurationInterface
+{
+    public function getConfigTreeBuilder(): TreeBuilder
+    {
+        $treeBuilder = new TreeBuilder('petit_press_gps_messenger', 'array');
```

Comment:
Is the second argument unnecessary? I can see it's "array" by default since SF4.4.

Reply:
Fixed, thanks.